site/docs/concepts/config-policies.md

```diff
@@ -215,57 +215,45 @@ config[{"capability": "copy", "decision": decision}] {
 ```
 Note that an empty ConfigDecisions map will be returned if the expiration date is exceeded by the time when the policy is applied. 
 
-## Taking infrastructure metrics into consideration
 
-When writing configuration policies, infrastructure metrics and costs may also be taken into account in order to optimize the generated data plane. 
-For example, selection of a storage account may be based on a storage cost, selection of a cluster may provide a restriction on cluster capacity, and so on. 
-Infrastructure attributes and metrics are stored in the `/tmp/adminconfig/infrastructure.json` directory of the manager pod. Collection of the metrics and their dynamic update is beyond the scope of Fybrik. One may develop or use 3rd party solutions for monitoring and updating these infrastructure metrics.
+## Optimization goals
```

Comment:
Not related to the current change but ...
I will continue reviewing later, but a couple of general comments:
* Should the title of this page be IT config policies, rather than config policies?
* Regarding the policySetID, I wasn't aware that we had a way to link a policy set to a particular FybrikApplication.  Why?  When does this make sense?  
* There are instructions for providing custom policies when deploying fybrik, but no mention of how to update them after it's deployed

Reply:
I'll change to IT config policies and will add instructions how to update the policies.
Regarding policySetID, when we presented policies to get feedback, one of the feedbacks was that different organizations may have different policies while all the policies are kept in the same repo, and this is the way to differentiate. This approach is half baked: there is a possibility to write policies with different ids but we don't populate this information in FybrikApplication, so for now all policies are global for all fybrik applications.

Reply:
OK.  I didn't recall that feedback.  In any case, I don't think it makes sense to expect the policySetID to be passed in the FybrikApplication.  I think this is a larger issue.  Perhaps just remove this section about policySetID and open a separate issue or discussion to decide how to address correctly?

Reply:
Done.